include/tvm/relay/dataflow_matcher.h

```diff
@@ -90,7 +91,8 @@ Expr RewritePatterns(Array<DFPatternCallback> callbacks, Expr expr);
  *
  * \return Return the paritioned Expr.
  */
-Expr PartitionPattern(DFPattern pattern, Expr expr);
```

Comment:
The history's a bit confusing here, did the last version not have an attrs argument?

Reply:
Yes, missed it in the last PR yesterday :(. Kind of surprised I didn't get a compiler warning.

Reply:
That is surprising, alright no worries then.